_tour/for-comprehensions.md

```diff
@@ -42,7 +42,7 @@ def foo(n: Int, v: Int) =
 
 foo(10, 10) foreach {
   case (i, j) =>
-    print(s"($i, $j) ")  // prints (1, 9) (2, 8) (3, 7) (4, 6) (5, 5)
+    println(s"($i, $j) ")  // prints (1, 9) (2, 8) (3, 7) (4, 6) (5, 5)
```

Comment:
the output would be 
```scala
(1, 9)
(2, 8)
(3, 7)
(4, 6)
(5, 5)
```

Reply:
true, but not sure the difference is worth the vertical space